schedule/reschedule.py

```diff
@@ -171,43 +174,37 @@ def apply_fuzz(self, ivl):
             return ivl
         min_ivl, max_ivl = get_fuzz_range(ivl, self.elapsed_days, self.maximum_interval)
         self.elapsed_days = 0
-        if not self.enable_load_balance:
-            if int_version() >= 231001:
-                return ivl + mw.col.fuzz_delta(self.card.id, ivl)
-            else:
-                return int(self.fuzz_factor * (max_ivl - min_ivl + 1) + min_ivl)
-        else:
-            # Load balance
-            due = self.card.odue if self.card.odid else self.card.due
-            last_review = get_last_review_date(self.card)
-
-            if self.apply_easy_days:
-                if due > last_review + max_ivl + 2:
-                    current_ivl = due - last_review
-                    min_ivl, max_ivl = get_fuzz_range(
-                        current_ivl, self.elapsed_days, current_ivl
-                    )
-
-            if last_review + max_ivl < self.today:
-                return min(ivl, max_ivl)
-
-            min_ivl = max(min_ivl, self.today - last_review)
-
-            possible_intervals = list(range(min_ivl, max_ivl + 1))
-            review_cnts = []
-            for i in possible_intervals:
-                check_due = last_review + i
-                if check_due > self.today:
-                    review_cnts.append(self.due_cnt_per_day[check_due])
-                else:
-                    review_cnts.append(self.due_today + self.reviewed_today)
-
-            best_ivl = self.load_balance(
-                possible_intervals,
-                review_cnts,
-                last_review,
+
+        # Load balance
+        due = self.card.odue if self.card.odid else self.card.due
+        last_review = get_last_review_date(self.card)
+
+        if due > last_review + max_ivl + 2:
```

Comment:
We want this to apply only when the user presses "Apply easy days now" or uses the easy days on specific days feature.

Otherwise, the user won't be able to correct the due date of cards whose due date is larger than the max limit of fuzz range.

Reply:
Done.